### PR TITLE
v0.16.2

### PR DESCRIPTION
**Pull request by ParthivNaresh**
v0.16.2 Apr 25, 2022

- Fixes
     - Fixed import issues regarding ``pyarrow`` and made ``python-dateutil>=2.8.1`` a required dependency #1397

### CODE REVIEW

**Review by gsheni — APPROVED** [review]

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -3,15 +3,21 @@
 Release Notes
 -------------
 
-Future Release
-==============
+.. Future Release
```
Comment (bchen1116) [inline docs/source/release_notes.rst:6]: do we always have the `..` and space here?

Comment (ParthivNaresh) [reply docs/source/release_notes.rst:6]: Yep! It's necessary as part of restructured text syntax for commenting: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#comments

**Review by jeremyliweishih — APPROVED** [review]
might need to look at the formatting for release notes but send it otherwise!

**Review by bchen1116 — APPROVED** [review]
!!

**Review by thehomebrewnerd — APPROVED** [review]

**Pull request merged**